lib/pg.ddl.js

```diff
@@ -391,9 +403,13 @@ const generateTable = (
   const [unresolved, resolvableLinks] =
     getResolvableLinks(name, unresolvedLinks, links, existingTables);
 
+  const pk = `pk${name}Id`;
+  validateIdentifier(pk, 'constraint');
+
   const primaryKey =
     `ALTER TABLE ${escapeIdentifier(name)} ` +
-    `ADD CONSTRAINT pk${name}Id PRIMARY KEY (${escapeIdentifier('Id')});`;
+    `ADD CONSTRAINT ${escapeIdentifier(pk)} ` +
+    `PRIMARY KEY (${escapeIdentifier('Id')});`;
```

Comment:
Why is `'Id'` escaped here using `escapeIdentifier`? It is not escaped on line
https://github.com/metarhia/globalstorage/blob/1386aced2697ab0c859808f9c808aacd1ac61d07/lib/pg.ddl.js#L231

Reply:
It is now escaped.